mautic/mautic
# Switch to skip empty decrypted keys instead of returning empty array and dropping other keys

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          |  yes
| New feature/enhancement? (use the a.x branch)      |  no
| Deprecations?                          |  no
| BC breaks? (use the c.x branch)        |  no
| Automated tests included?              |  no change
| Related user documentation PR URL      |  -
| Related developer documentation PR URL |  -
| Issue(s) addressed                     | Fixes #10573

#### Description:

Changes slightly the behavior of AbstarctIntegration::decryptApiKeys in order to not drop all settings when one setting is set to false. 

Note: This might also apply to previous versions - 3.x etc.

#### Steps to test this PR:

See #10573




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10574"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
